src/extensions/husky-setup-extension.test.js

```diff
@@ -1,61 +1,307 @@
-const extend = require('./husky-setup-extension');
-const { createToolboxMock, createExtensionInput } = require('../utils/test/mocks');
+const extend = require('./husky-setup-extension')
+const {
+  createToolboxMock,
+  createExtensionInput,
+} = require('../utils/test/mocks')
 
 describe('husky-setup-extension', () => {
   it('should be defined', () => {
-    expect(extend).toBeDefined();
-  });
+    expect(extend).toBeDefined()
+  })
 
   it('should set setupHusky on toolbox', () => {
-    const toolbox = {};
-    extend(toolbox);
-    expect(toolbox.setupHusky).toBeDefined();
-  });
+    const toolbox = {}
+    extend(toolbox)
+    expect(toolbox.setupHusky).toBeDefined()
+  })
 
   describe('setupHusky', () => {
-    const toolbox = createToolboxMock();
+    const toolbox = createToolboxMock()
 
     beforeAll(() => {
-      extend(toolbox);
-    });
+      extend(toolbox)
```

Comment:
I if we refactor to remove the repeating declarations we can get rid of this `beforeAll` block too